.github/workflows/test.yml

```diff
@@ -14,6 +14,7 @@ on:
       # to whitelist those branches for CI run here.
       # See also https://greenkeeper.io/faq#no-initial-pr
       - 'greenkeeper**'
+      - 'seanyen**'
```

Comment:
@thomas-moulard this is an interesting way to get CI running from forks :)

Reply:
Sorry I forgot to remove that. 😅